.github/workflows/presubmit.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
     - uses: ./.github/actions/install-deps
       with:
         k8sVersion: ${{ matrix.k8sVersion }}
+    - name: Enable the actionlint matcher
+      run: echo "::add-matcher::.github/actionlint-matcher.json"
```

Comment:
Can we not capture this inside `make verify`?

Reply:
Not really necessary. This is _super_ specific to GHA so having this generalized feels unnecessary to me